src/nodes.coffee

```diff
@@ -2245,7 +2240,14 @@ UTILITIES =
       return child;
     }
   "
-
+  # Creates and applies arguments to a class constructor.
+  applyCtor: (ctor, args) -> "
+    function(ctor, args) {
+      var child = Object.create(ctor.prototype);
```

Comment:
your right but would be a shame to be held back by obsolete browsers - http://kangax.github.io/compat-table/es5/